README.md

```diff
@@ -166,6 +166,9 @@ on [Remote Monitoring Multiple Harvesters](https://github.com/martomi/chiadog/wi
 You can enable more verbose logging from `config.yaml` by changing `INFO` to `DEBUG`. You should see logs for every
 keep-alive event from the harvester.
 
+You can also set the `debug_mode` to `true` in `config.yaml` under `daily_stats`. In this mode `chiadog` will send
```

Comment:
I would contain this to PR #176 
Also, if you agree to my suggestion, this would be omitted in favor of the argparser approach

Reply:
👍 Will drop that change from README